src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -0,0 +1,236 @@
+<!--
+ - @copyright Copyright (c) 2020 Georg Ehrke <oc.list@georgehrke.com>
+ -
+ - @author 2020 Georg Ehrke <oc.list@georgehrke.com>
+ -
+ - @license GNU AGPL version 3 or any later version
+ -
+ - This program is free software: you can redistribute it and/or modify
+ - it under the terms of the GNU Affero General Public License as
+ - published by the Free Software Foundation, either version 3 of the
+ - License, or (at your option) any later version.
+ -
+ - This program is distributed in the hope that it will be useful,
+ - but WITHOUT ANY WARRANTY; without even the implied warranty of
+ - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ - GNU Affero General Public License for more details.
+ -
+ - You should have received a copy of the GNU Affero General Public License
+ - along with this program. If not, see <http://www.gnu.org/licenses/>.
+ -
+ -->
+<docs>
+
+	### General description
+
+	This component allows the user to pick an emoji.
+
+	### Usage
+
+	* Listen to the select event and pass in an HTML element that will be treated as a trigger:
+
+	```vue
+	<template>
+		<div>
+			<EmojiPicker @select="log">
+				<button> Click Me </button>
+			</EmojiPicker>
+		</div>
+	</template>
+	<script>
+		export default {
+			methods: {
+				log(e) {
+					console.debug(e)
+				},
```

Comment:
Maybe the example could be easier than outputting to log?
Maybe adding a line with `selected emoji: 🦀` or something?